.github/workflows/python_build.yml

```diff
@@ -77,6 +77,9 @@ jobs:
       - name: Build Python package
         run: maturin build --release --no-sdist --strip
 
+      - name: Check that the lock file is up to date
+        run: git diff --exit-code Cargo.lock
```

Comment:
can you use `--frozen` and `--locked` per [doc][1]

[1]: https://doc.rust-lang.org/cargo/commands/cargo-install.html

Reply:
Yep, great idea. Thanks